core/src/commands/get/get-test-result.ts

```diff
@@ -12,10 +12,24 @@ import { printHeader } from "../../logger/util"
 import chalk from "chalk"
 import { getArtifactFileList, getArtifactKey } from "../../util/artifacts"
 import { joi, joiArray } from "../../config/common"
-import { getTestActionFromArgs, runTestArgs } from "../run/run-test"
 import { GetTestResult } from "../../plugin/handlers/test/get-result"
-
-const getTestResultArgs = runTestArgs
+import { ParameterValues, StringParameter } from "../../cli/params"
+import { ParameterError } from "../../exceptions"
+import { ConfigGraph } from "../../graph/config-graph"
+import { GardenModule, moduleTestNameToActionName } from "../../types/module"
+import { findByName, getNames } from "../../util/util"
+
+const getTestResultArgs = {
+  name: new StringParameter({
+    help:
+      "The test to run. If using modules, specify the module name here and the test name from the module in the second argument",
```

Comment:
```suggestion
      "The name of the test. If this test belongs to a module, specify the module name here instead, and specify the test name from the module in the second argument.",
```
Suggesting this change since here we're fetching the result, not running the test.